src/pages/pricing-plans/PricingPlans.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="text-textPrimary">
+    <h1 class="text-[32px] md:text-5xl leading-[110%] md:leading-[56px] font-bold">
+      Choose your plan
+    </h1>
+    <div class="py-4 text-lg leading-[26px]">
+      If you need more info about our pricing, please check <span class="text-primary underline">Pricing Guidelines</span>.
+    </div>
+    <div class="flex flex-col p-4 bg-backgroundSecondary">
+      <div class="flex justify-center">
+        <div class="space-x-[1px] p-[2px] border rounded">
+          <va-button 
+            v-for="option in ['Monthly', 'Annual']"
+            class="w-[85.5px]"
+            color="textPrimary"
+            preset="plain"
+            :pressedOpacity="0.85"
+            :hoverOpacity="0.85"
+            :style="{ ...durationButtonStyles, ...( planDuration === option && { '--va-0-background-color': '#ECF0F1' } ) }"
+            @click="planDuration = option"
+          >
+            {{ option }}
+          </va-button>
+        </div>
+      </div>
+      <div class="flex flex-col md:flex-row justify-center items-center space-y-4 md:space-x-6 md:space-y-0 mt-6">
+        <va-card 
+          v-for="plan in pricingPlans"
+          class="flex w-[326px] md:w-[349px] h-fit p-6 rounded-[13px]"
+          :class="plan.class"
+        >
+          <div 
+            class="space-y-8 md:space-y-10"
+            :class="{ '!space-y-10' : plan.model === 'Advanced' }"
+          >
+            <div class="space-y-4 text-center">
+              <h2 class="text-[28px] md:text-[32px] leading-10 font-bold">
+                {{ plan.title }}
+              </h2>
+              <va-badge 
+                v-for="b in plan.badges"
```

Comment:
It's considered a good practice to use abbreviations sparingly

So would recommend `badge` here